central/networkgraph/flow/datastore/internal/store/postgres/tests/store_test.go

```diff
@@ -75,33 +75,37 @@ func (s *NetworkflowStoreSuite) TestStore() {
 	secondCluster := fixtureconsts.Cluster2
 	store2 := postgresFlowStore.New(s.pgDB.DB, secondCluster)
 
+	now := time.Now().UTC()
+	timeNow := timestamp.FromGoTime(now)
+
+	time1 := timeNow.Add(time.Second)
+
 	networkFlow := &storage.NetworkFlow{
 		Props: &storage.NetworkFlowProperties{
 			SrcEntity:  &storage.NetworkEntityInfo{Type: storage.NetworkEntityInfo_DEPLOYMENT, Id: "a"},
 			DstEntity:  &storage.NetworkEntityInfo{Type: storage.NetworkEntityInfo_DEPLOYMENT, Id: "b"},
 			DstPort:    1,
 			L4Protocol: storage.L4Protocol_L4_PROTOCOL_TCP,
 		},
-		LastSeenTimestamp: protocompat.GetProtoTimestampFromSeconds(1),
+		LastSeenTimestamp: time1.Protobuf(),
```

Comment:
Is this actually the same time?  Why change from the protocompatiblity functions?